src/components/icons/assistant.tsx

```diff
@@ -2,12 +2,14 @@ import { ReactSVG, SVGProps } from 'react';
 
 interface AssistantIconProps {
 	size?: number;
+	className?: string;
 }
 
 export function AssistantIcon( {
 	size = 14,
 	'aria-hidden': ariaHidden,
-}: AssistantIconProps & SVGProps< ReactSVG > ) {
+	className,
+}: AssistantIconProps & SVGProps< ReactSVG > & { className?: string } ) {
```

Comment:
I wonder if we need to include `{ className?: string }` in the return type. The `className` should be already part of the `SVGAttributes`.

Reply:
Removed in https://github.com/Automattic/studio/pull/266/commits/1c92c8936197920e698d661c2af55dcf0e5cb42e